src/common/paths.ts

```diff
@@ -2,12 +2,14 @@ export const PathsV1 = {
   accounts: {
     root: "/api/v1/accounts",
     publicKey: "/api/v1/accounts/public_key",
+    subscription: "/api/v1/accounts/subscription",
   },
   admin: {
     root: "/api/v1/admin",
     accounts: {
       root: "/api/v1/admin/accounts",
-      subscriptions: "/api/v1/admin/accounts/subscription",
+      subscription: "/api/v1/admin/accounts/subscription",
+      subscriptionById: "/api/v1/admin/accounts/subscription/:did",
```

Comment:
maybe `subscriptionByDid` to avoid confusion with a uuid/oid which might be just called Id?